include/cityhash.h

```diff
@@ -32,6 +32,14 @@
 extern "C" {
 #endif
 
+/*
+ * We have 2/3-argument specialized versions of cityhash4,
+ * which can reduce instruction count and stack usage on some 32-bit arches.
```

Comment:
```suggestion
 * which can reduce instruction count, especially on 32-bit arches.
```
Stack size is not so critical here, since the functions are leaf and not inlined.